app/src/main/java/nerd/tuxmobil/fahrplan/congress/details/SessionDetailsViewModel.kt

```diff
@@ -13,6 +15,7 @@ import kotlinx.coroutines.launch
 import nerd.tuxmobil.fahrplan.congress.alarms.AlarmServices
 import nerd.tuxmobil.fahrplan.congress.models.Session
 import nerd.tuxmobil.fahrplan.congress.navigation.RoomForC3NavConverter
+import nerd.tuxmobil.fahrplan.congress.notifications.NotificationHelper
```

Comment:
:broom: Please check the diff of the files you touched before you push for unused code.